fov_processing_pipeline/data/quilt.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
New function for data read in via quilt. Looks good. Don't have the skills (yet?) to see 'through' the code, spot error, etc.